client/questions_answers/QuestionsAnswers.jsx

```diff
@@ -25,24 +44,35 @@ class QuestionsAnswers extends React.Component {
         }
         if (!nextTwoQuestions.length) {
           this.setState({
+            showSearch: true,
             questions: firstTwoQuestions
           });
           return;
         }
 
         this.setState({
           showMoreAnsweredQuestionsButton: true,
+          showSearch: true,
           nextTwoQuestions: nextTwoQuestions,
           questions: firstTwoQuestions
         });
 
       });
-
-    this.updateQuestionsList = this.updateQuestionsList.bind(this);
```

Comment:
Why did you include this here vs in the constructor?

Reply:
Lets discuss tonight in standup...

Reply:
Sounds good!